### ISSUE TITLE
update outdated dependencies

### ISSUE THREAD

**Issue by cnishina** [angular/protractor#3249]
It appears we have some outdated packages:

```
>: npm outdated
Package           Current  Wanted  Latest  Location
body-parser        1.14.2  1.14.2  1.15.1  body-parser
chai                3.4.1   3.4.1   3.5.0  chai
chai-as-promised    5.2.0   5.2.0   5.3.0  chai-as-promised
glob                6.0.4   6.0.4   7.0.3  glob
jshint              2.9.1   2.9.1   2.9.2  jshint
mocha               2.3.4   2.3.4   2.5.3  mocha
request            2.67.0  2.67.0  2.72.0  request
saucelabs           1.0.1   1.0.1   1.2.0  saucelabs
typescript          1.8.2   1.8.2  1.8.10  typescript
typings             0.8.1   0.8.1   1.0.4  typings
```

To upgrade these, make sure the entire [protractor test suite passes](https://github.com/angular/protractor/blob/master/CONTRIBUTING.md#testing-your-changes). 

Also, typings version 1.04 will require also editing the typings.json and tsconfig.json. See my fork of webdriver-manager for a similar change to [typings.json](https://github.com/cnishina/webdriver-manager/blob/pr_typings_update/typings.json) and [tsconfig.json](https://github.com/cnishina/webdriver-manager/blob/pr_typings_update/tsconfig.json).

**Comment by cnishina** [comment]
@juliemr Thoughts on not updating any of these packages?

**Comment by juliemr** [comment]
Nope, I think these are all OK.

**Comment by cnishina** [comment]
@manoj9788 Thanks for getting this started.

**Comment by cnishina** [comment]
Closed with 43530693f6cafd1d3cd3407bd5d1088b51ab8101
